deployment/helm/balloons/templates/daemonset.yaml

```diff
@@ -91,11 +94,15 @@ spec:
         hostPath:
           path: /var/run/nri
           type: DirectoryOrCreate
-      {{- if .Values.nri.patchContainerdConfig }}
+      {{- if .Values.nri.patchRuntimeConfig }}
       - name: containerd-config
         hostPath:
           path: /etc/containerd/config.toml
-          type: File
+          type: FileOrCreate
+      - name: crio-config
+        hostPath:
+          path: /etc/crio/crio.conf.d/
+          type: Directory
```

Comment:
`DirectoryOrCreate`? Doesn't container creation fail otherwise if this directory does not exist?

Reply:
Yes it will. fixed.